Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -8,6 +9,22 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
     /// - SeeAlso: https://datatracker.ietf.org/doc/html/rfc3986#section-2.2
     fileprivate static let forbiddenUserInfoCharacters = [":", "/", "?", "#", "[", "]", "@"]
 
+    private struct Names {
```

Comment:
introduced a struct to store the name constants to avoid name collision with the `MongoConnectionString` fields